app/graphql/types/query_type.rb

```diff
@@ -19,5 +28,16 @@ def authorities
     def elections
       Election.all
     end
+
+    field :pending_message,
+          Types::PendingMessageType,
+          null: true,
+          description: "Returns the information for a given message" do
+      argument :id, ID, required: true
+
+      def resolve(_parent, args, _context)
+        PendingMessage.find(args[:id])
```

Comment:
The `find` method raises an exception if the `id` doesn't exist so this type cannot be resolved to `null` at the moment. I suggest using `where(id: args[:id]).first` or just `find_by_id` 👍